script/int-log-compiler.sh

```diff
@@ -54,6 +54,23 @@ while test $# -gt 0; do
     shift
 done
 
+sanity_test ()
+# Verify the running shell and OS environment is sufficient to run these tests.
+{
+# Check special file
+if [ ! -e /dev/urandom ]; then
+	echo  "Missing file /dev/urandom; exiting"
+       	exit 1
```

Comment:
Your indentation is inconsistent. It may not be visible in your editor though since you're mixing tabs and spaces (please don't do that either).

Reply:
this comment applies to the rest of this function

Reply:
Accepted, as above.  One line was copied and pasted and I added the other line, which is why it is inconsistent.